### PR TITLE
feat(HighContrast): Expose the XAML HighContrastChanged event to JS

### PR DESCRIPTION
**Pull request by antonkh**
The event is exposed as highContrastDidChange in AccessibilityInfo. For now
it only tells if high contrast is enabled or disabled in system settings. Later it
can tell what are the chosen high contrast colors: Windows 10 allows to create
custom high contrast themes with user chosen colors.

### CODE REVIEW

**Comment by rozele** [comment]
@antonkh does the HighContrastChanged event have to be subscribed on the Dispatcher thread? Would it be possible to move this into a custom native module for accessibility?

File: ReactWindows/ReactNative/Modules/AccessibilityInfo/AccessibilityInfoModule.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        private readonly AccessibilitySettings _accessibility = new AccessibilitySettings();
+
+        public AccessibilityInfoModule(ReactContext reactContext)
+            : base(reactContext)
+        {
+            _accessibility.HighContrastChanged += (sender, args) =>
+            {
+                Context.GetJavaScriptModule<RCTDeviceEventEmitter>()
+                    .emit("highContrastDidChange", sender.HighContrast);
+            };
```
Comment (rozele) [inline ReactWindows/ReactNative/Modules/AccessibilityInfo/AccessibilityInfoModule.cs]: Can we set up and tear down this event in `Initialize` and `OnReactInstanceDispose`?

**Review by rozele — CHANGES_REQUESTED** [review]
:clock1:

File: ReactWindows/ReactNative/Modules/AccessibilityInfo/AccessibilityInfoModule.cs (hunk 1 above)
Comment (antonkh) [reply ReactWindows/ReactNative/Modules/AccessibilityInfo/AccessibilityInfoModule.cs]: Fixed

**Pull request merged**